tensorbay/opendataset/CCPD/catalog.json

```diff
@@ -0,0 +1,34 @@
+{
+    "POLYGON": {
+        "attributes": [
+            {
+                "name": "blurriness",
+                "type": "integer",
+                "minimum": 0,
+                "maximum": 130
```

Comment:
do we have `description` for these attributes?

Reply:
we only have some simple descriptions, such as:
brightness: The brightness of the license plate region. 
blurriness: The Blurriness of the license plate region.